Add sepolicy for hwcomposer and gralloc

renchenglei opened the pull request:
We have some adjustments for the property setting for
hwcomposer and gralloc. Here, we add the sepolicy change
for them

Tracked-On: OAM-91001
Signed-off-by: ji, zhenlong z <zhenlong.z.ji@intel.com>
Signed-off-by: Ren Chenglei <chenglei.ren@intel.com>

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

Cactus automatically verified this PR.

For more information, see: /absp/builders/celadon-autobuild/builds/2261

renchenglei committed:
Add sepolicy for hwcomposer and gralloc

We have some adjustments for the property setting for
hwcomposer and gralloc. Here, we add the sepolicy change
for them

Tracked-On: OAM-91001
Signed-off-by: ji, zhenlong z <zhenlong.z.ji@intel.com>
Signed-off-by: Ren Chenglei <chenglei.ren@intel.com>

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

Cactus automatically verified this PR.

For more information, see: /absp/builders/celadon-autobuild/builds/2287

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/2076

renchenglei commented:
Hi @kwang13, could you help take a review of this changes? thanks a lot!

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/2077

Done by "Ren, Chenglei" <chenglei.ren@intel.com>

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon-premerge/builds/1061

Done by "Ren, Chenglei" <chenglei.ren@intel.com>

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by "Ren, Chenglei" <chenglei.ren@intel.com> 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon-mergerequest/builds/1771
